### PR TITLE
[Nataliia Falinska] Tasks 11_20

### PR DESCRIPTION
**Pull request by NFalinska**

### CODE REVIEW

File: tasks/12_title-case/title-case.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */ 
+
+export function titleCase(str) {
+  let newStr = str.toLowerCase()
+      .split(' ')
+      .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+      .join(' ');
+  return newStr;
```
Comment (IgorStetsiuk) [inline tasks/12_title-case/title-case.js]: You don't need the redundant variable here. Please refer to previous PR https://github.com/griddynamics/ui-internship-lwo2/pull/8.

File: tasks/14_clone-arr/clone-arr.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */ 
+
+export function cloneArr(arr) {
+  let clone = arr.slice(0);
+  return clone;
```
Comment (IgorStetsiuk) [inline tasks/14_clone-arr/clone-arr.js]: Same as above. Why did you create this `clone` variable? You don't need it.

File: tasks/13_largest-of-four/largest-of-four.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */ 
+
+export function largestOfFour(arr) {
+  return arr.map(function(subArr) {
```
Comment (IgorStetsiuk) [inline tasks/13_largest-of-four/largest-of-four.js]: Can be write in more readable form `arr.map(subArr => Math.max.apply(null, subArr)); `
Good solution!

File: tasks/16_repeat-string/repeat-string.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */ 
+
+export function repeatStringNumTimes(str, num) {
+  let newStr = '';
+  for (let i=0; i<num; i++) {
```
Comment (IgorStetsiuk) [inline tasks/16_repeat-string/repeat-string.js]: Spacing here.  `let i = 0; i < num; i++`

File: tasks/20_truncate-array/truncate-array.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */ 
+
+export function truncateArray(arr, num) {
+  let newArr = arr.splice(num);
```
Comment (IgorStetsiuk) [inline tasks/20_truncate-array/truncate-array.js]: Same as above. No need that var

File: tasks/11_find-longest-word/find-longest-word.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */ 
+
+export function findLongestWord(str) {
+  let strArr = str.split(' ');
+  let theLongest = strArr[0];
+  for (let i=0; i<strArr.length; i++) {
```
Comment (tarasbobak) [inline tasks/11_find-longest-word/find-longest-word.js]: use proper spacing please
for (let i = 0; i < strArr.length; i++) {

File: tasks/11_find-longest-word/find-longest-word.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+    if (strArr[i].length>theLongest.length) {
```
Comment (tarasbobak) [inline tasks/11_find-longest-word/find-longest-word.js]: SAme here if (strArr[i].length > theLongest.length) {

File: tasks/12_title-case/title-case.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */ 
+
+export function titleCase(str) {
```
Comment (tarasbobak) [inline tasks/12_title-case/title-case.js]: Implementation is very good

File: tasks/13_largest-of-four/largest-of-four.js (hunk 3 above)
Comment (tarasbobak) [reply tasks/13_largest-of-four/largest-of-four.js]: Good catch Ihor!

File: tasks/14_clone-arr/clone-arr.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */ 
+
+export function cloneArr(arr) {
+  let clone = arr.slice(0);
```
Comment (tarasbobak) [inline tasks/14_clone-arr/clone-arr.js]: return arr.slice(); should be best fit for this task

File: tasks/17_truncate-string/truncate-string.js [hunk 10]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */ 
+
+export function truncateString(str, stop) {
+  if (str.length > stop) {
+    return str.slice(0, stop > 3 ? stop-3 : stop) + '...';
```
Comment (tarasbobak) [inline tasks/17_truncate-string/truncate-string.js]: Good solution

File: tasks/18_humanize-format/humanize-format.js [hunk 11]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */ 
+
+export function humanizeFormat(date) {
```
Comment (tarasbobak) [inline tasks/18_humanize-format/humanize-format.js]: Yeah, it is a solution that will pass tests, but it is not full.
Consider ending is 13. It should not have 'rd', but 'th'.

**Comment by tarasbobak** [comment]
Overly good job! Honestly saying I would request changes to remove these 'ignore eslint' comments. But I'll skip it for the last time.

Please do not do the same mistakes in the future!

**Pull request merged**